.github/workflows/pythonapp.yml

```diff
@@ -27,6 +27,10 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install -r requirements.txt
+#       Temporarily use satyaog repo until it gets merged into mila-iqia
+        wget -q --output-document - \
+          https://raw.githubusercontent.com/satyaog/covid_p2p_risk_prediction/inference-server/requirements.txt | \
```

Comment:
using mine until [it gets merge](https://github.com/mila-iqia/covid_p2p_risk_prediction/pull/14) on mila-iqia